### ISSUE TITLE
Allow colcon release-in-container to use more that 1 cpu

### ISSUE THREAD

**Issue by aosmw** [canonical/colcon-in-container#24]
I noticed that the `colcon release-in-container` does a build that only uses 1 cpu.

Here is a related discussion that bloom is that way in order to make it work for the ros build farm use case.
https://github.com/ros-infrastructure/bloom/pull/643

**Comment by Guillaumebeuzeboc** [comment]
Thanks for the feedback and the link.
In the case of the `release-in-container` we are building the packages in a deterministic order, so that shouldn't cause any issue.
Now I am more wondering if we should and how to integrate such a change in `release-in-container`.

I would probably be simpler if `Bloom` could get released with that change, since the project is not meant to diverge from upstream.

**Comment by aosmw** [comment]
The change, IF you decided to do it, could be allow debian/compat to be overwritten to a number (10 or even 13).  other than what it currently defaults to after the `bloom generate` call.  

Here is a script where it is done with a simple `echo 11 > debian/compat`  which is a comment in 
https://github.com/ros-industrial/industrial_ci/issues/697#issuecomment-876293987

Turns out that 11 is not recommended.  10 would be enough to satisfy this issue.  Automatically do the build in parallel.

**Comment by artivis** [comment]
I agree that I would prefer this to land in bloom rather than as a workaround here.

Linking the related bloom ticket: https://github.com/ros-infrastructure/bloom/pull/643.
